src/linker/steps/dev/python_pandas/dummy_step.py

```diff
@@ -25,9 +26,9 @@ def load_file(file_path, file_format=None):
 diagnostics = {}
 
 if "DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS" in os.environ:
-    main_input_file_paths = os.environ["DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS"].split(",")
+    main_input_file_paths = json.loads(os.environ["DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS"])
```

Comment:
this didn't work unless i did json or ast.literal_eval or similar. Im not sure what the process is for re-building the stored containers?

Reply:
I don't really understand this change, what was failing with the old code?

Reply:
Also, how did you test this without re-building? I believe I've done most (all?) of the dummy container building, but there are instructions for how to do it along with the containers themselves in the research repo.

Reply:
IIRC what was happening was that since the environment variables were (have to be?) encoded as strings, the brackets get encoded as well, e.g. "['/foo.csv', '/bar.csv']" (or maybe swap the single and double quotes) and splitting by comma leaves you with the unwanted brackets. The easiest/safest thing imo was to encode it in JSON format and decode it in the notebook.

Reply:
I just built new images and containers in a temp directory and pointed to them with the metadata yaml